main.js

```diff
@@ -160,4 +160,174 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+// Opdracht 1a
+let tvNames = []
+inventory.map(item => {
+    tvNames.push(item.name)
+})
+
+console.log(tvNames)
+
+// Opdracht 1b
+let soldOut = []
+
+inventory.filter(tv => {
+    if (tv.originalStock - tv.sold <= 0) {
+        soldOut.push(tv)
+    }
+})
+
+console.log(soldOut)
+
+// Opdracht 1c
+let ambiLight = []
+
+inventory.filter(tv => {
+    if (tv.options.ambiLight === true) {
+        ambiLight.push(tv)
+    }
+})
+
+console.log(ambiLight)
+
+// Opdracht 1d
+// inventory.sort((a, b) => {
+//     return a.price - b.price
+// })
+
+console.log(inventory)
+
+// Opdracht 2a
+let sold = inventory.reduce((acc, item) => {
+    return acc + item.sold;
+}, 0);
+
+console.log(sold);
+
+// Opdracht 2b
+const locationSold = document.getElementById('sold');
+locationSold.innerHTML = `Amount of TV's sold: <span class="green-text">${sold}</span`;
+
+// Opdracht 2c
+let bought = inventory.reduce((acc, item) => {
+    return acc + item.originalStock
+}, 0)
+
+console.log(bought)
+
+// Opdracht 2d
+const locationBought = document.getElementById('bought');
+locationBought.innerHTML = `Amount of TV's bought: <span class="blue-text">${bought}</span`;
+
+// Opdracht 2e
+const locationInStock = document.getElementById('in-stock');
+locationInStock.innerHTML = `Amount of TV's in stock: <span class="red-text">${bought - sold}</span>`;
+
+// Opdracht 3a & b
+
+function brands(inventory) {
+    let brands = []
+
+    inventory.map(brand => {
+        brands.push(brand.brand)
+    })
```

Comment:
Voor de map functie ook
```suggestion
    const brands = inventory.map(brand => brand.brand);
```